src/Microsoft.AspNet.SignalR.Core/Infrastructure/IMemoryPool.cs

```diff
@@ -6,11 +6,14 @@ namespace Microsoft.AspNet.SignalR.Infrastructure
 {
     public interface IMemoryPool
     {
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
```

Comment:
@davidfowl - is there any reason why we need this as property on the interface instead of using empty array directly in the code?